image-layout.md

```diff
@@ -199,7 +200,11 @@ Those tags will often be represented in an image-layout repository with matching
 }
 ```
 
-This illustrates an index that provides two named manifest references and an auxiliary mediatype for this image layout.
+This illustrates an index that provides two named references and an auxiliary mediatype for this image layout.
 
+The first named reference (`"stable-release"`) points to another index that might contain multiple references with several platform specification and annotations.
```

Comment:
Backticks *and* quotes seems like overkill.  Can we use `stable-release` here (and `v1.0` below)?  Those are the names; the quotes are just part of representing strings in JSON.

Also probably “several platform sepecification” → “distinct platforms” or some such.